### ISSUE TITLE
Build GraffAPI driver

### ISSUE THREAD

**Issue by GearsAD** [JuliaRobotics/DistributedFactorGraphs.jl#94]

**Comment by Affie** [comment]
What about making it a separate package? It will likely have higher level functionality in and having a separate package might be easier to maintain and test. Especially when it comes to dependencies. This way GraffAPI/GraffSDK can depend on RoME, IIF, etc.

**Comment by dehann** [comment]
just the next version of this?
https://github.com/GearsAD/GraffSDK.jl

**Comment by GearsAD** [comment]
Yeah, in this case we would have a mapping from Graff to DFG so you it would be able to use 'addvariable' via DFG normally and it would call the REST API, using security etc. Could do as separate package.
